bundle/Resources/config/templating.yml

```diff
@@ -66,7 +66,6 @@ services:
     # Overriding core helper
     ezpublish.templating.global_helper:
         alias: ezpublish.templating.global_helper.legacy
```

Comment:
```suggestion
        alias: ezpublish.templating.global_helper.legacy
        # Needs to be public to use the service as a Twig global variable
        public: true
```

Reply:
@andrerom Here you go. Care should be taken to not end up with duplicate `public` option, as I couldn't add the suggestion to a deleted line.

Reply:
@xserna Could you update PR with suggested change above?

Reply:
> @xserna Could you update PR with suggested change above?

Sure, I've this on my todo list. Sorry for the delay, I'll update it now.